domain/infosync/mock_info.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package infosync
+
+import (
+	"sync"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/config"
+	"github.com/pingcap/tidb/parser/mysql"
+	"github.com/pingcap/tidb/sessionctx/binloginfo"
+	"github.com/pingcap/tidb/util/versioninfo"
+)
+
+// MockGlobalServerInfoManagerEntry is a mock global ServerInfoManager entry.
+var MockGlobalServerInfoManagerEntry = &MockGlobalServerInfoManager{
+	mockServerPort: 4000,
```

Comment:
If this is a port that must be used, please randomize it.

```
mockServerPort: 0,
```

Reply:
> If this is a port that must be used, please randomize it.
> 
> ```
> mockServerPort: 0,
> ```

Currently it's just for identification of dist tasks, if we use the port to send rpc, I will find a better way to generate the port